tactics.py

```diff
@@ -0,0 +1,108 @@
+denaro = "*"
+settebello_symbol = "07"+denaro
+lay_card_button_name = "Połóż kartę"
+claim_cards_button_name = "Zbierz karty"
+opponent_hand_frame_name = "Ręka przeciwnika"
+actions_frame_name = "Akcje"
+my_hand_frame_name = "Moja ręka"
+table_frame_name = "Stolik"
+this_was_scopa = "To była scopa!"

```

Comment:
1. Constant names should be written in CAPS.
2. UI labels could be stored in a dictionary (let's say `LABELS`) in order to minimize namespace clutter.
3. UI labels would make much more sense to be stored in a file/class that actually renders the UI, instead of a file that stores utility/helper functions related to the game logic.